testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-delta-lake-databricks/delta.properties

```diff
@@ -1,5 +1,6 @@
 connector.name=delta-lake
-hive.metastore.uri=${ENV:HIVE_METASTORE_URI}
```

Comment:
Before the change we were testing against OSS Delta Lake using HMS as the support.
Now it seems we're using Glue? or, was this file not used before?

Reply:
This file was never used before for testing the `trino-delta-lake` connector functionality.

Reply:
We're using HMS for testing against open source Delta and Glue for Databricks Delta. Is that right?

Reply:
We're using HMS for `singlenode-delta-lake-oss` environment and Glue for `singlenode-delta-lake-databricks73` and `singlenode-delta-lake-databricks91` environments.